craterpy/exceptions.py

```diff
@@ -0,0 +1,17 @@
+"""Custom exceptions used by this project."""
```

Comment:
Not sure about the goal of using custom exceptions. If we want to handle all craterpy exceptions, then they should subclass a common base class instead of different parents.

Reply:
Good point, I will look into this in future, but right now a catch-all is probably unnecessary